extensions/scanning/schema.json

```diff
@@ -73,6 +76,36 @@
           "type": "string",
           "format": "date-time",
           "pattern": "(\\+00:00|Z)$"
+        },
+        "summaries": {
+          "type": "object",
+          "properties": {
+            "scan:is_original": {
+              "type": "array",
+              "minItems": 1,
+              "items": {
+                "type": "boolean"
+              }
+            },
+            "scan:scanned": {
+              "type": "object",
+              "required": ["minimum", "maximum"],
+              "properties": {
+                "minimum": {
+                  "title": "earliest date",
+                  "type": ["string", "null"],
+                  "format": "date-time",
+                  "pattern": "(\\+00:00|Z)$"
+                },
+                "maximum": {
+                  "title": "most recent date",
+                  "type": ["string", "null"],
+                  "format": "date-time",
+                  "pattern": "(\\+00:00|Z)$"
```

Comment:
Does this actually allow `null` though? I think you might need something which is explicitly optional, like

```
{
  "oneOf": [
    {
      "type": "string",
      "format": "date-time",
      "pattern": "(\\+00:00|Z)$"
    },
    {
      "type": "null"
    }
  ]
}
```

Reply:
hmm I'm not sure why I allowed the option of null in the first place, so have removed it. :thinking: